src/tests/intg/test_ssh_pubkey.py

```diff
@@ -261,12 +283,40 @@ def add_user_with_many_keys(request, ldap_conn):
     ent_list.add_user("user1", 1001, 2001, sshPubKey=pubkey_list)
     create_ldap_fixture(request, ldap_conn, ent_list)
 
-    conf = format_basic_conf(ldap_conn, SCHEMA_RFC2307_BIS)
+    config.PAM_CERT_DB_PATH = os.environ['PAM_CERT_DB_PATH']
+    conf = format_basic_conf(ldap_conn, SCHEMA_RFC2307_BIS, config)
     create_conf_fixture(request, conf)
     create_sssd_fixture(request)
     return None
 
 
+@pytest.fixture
+def add_user_with_ssh_cert(request, ldap_conn):
+    # Add a certificate to ldap, to manually test a cert from a smartcard.
+    # Export it and insert below.
```

Comment:
What do you mean with the second line of the comment?

Reply:
Removed since it is not related tonthis problem.